practice/src/javascripts/main.js

```diff
@@ -0,0 +1,31 @@
+import ProductList from './products/productList';
+import ProductTemplate from './templates/productTemplate';
+import ProductService from './services/productService';
+
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
you are using `<script src="../src/javascripts/main.js" type="module"></script>`, so we don't need to listen `DOMContentLoaded`